### DISCUSSION TITLE
Global change of double quote to single quote

### DISCUSSION THREAD

**Discussion by mogplus88** [vkbo/novelWriter#1575] [Help] [answered]
Is it possible to do a global change of double quote to single quote? I've written quite a bit of stuff using double quotes for dialog, and I've now read that single quotes are preferred. Changing them all manually is a long and arduous task. Is there an easier way?
Thanks.

**Comment by vkbo** [comment, marked as answer]
No, unfortunately not. You can use the search/replace feature and turn on "Search Next File", start on the first and then work your way through. It's not exactly efficient, but if you use regex, you can replace both quote symbols at the same time.

Assuming you're using the standard English double quotes, turn on RegEx in the search tool, and add the symbols with a pipe between them, like `“|”`, and in the replace box just enter the single straight quite from the keyboard. That will trigger the auto-replace for the proper symbol when you hit replace:

![image](https://github.com/vkbo/novelWriter/assets/1619840/7ae9ec48-1c19-47c5-a176-57cc1ab87bb6)

Then you can just hit the replace button, or press `Ctrl+Shift+1` until you're through the entire project, or however far you want to go.

**Reply by mogplus88** [reply]
Thanks, that worked. When I figured out that I had to copy and paste the double quotes from the text. Entering it from the keyboard didn't work, so I guess they get converted to opening double quote and closing double quote somehow. Now all I need is a 'replace all' option so I don't get rsi from clicking the replace button. :)

**Reply by vkbo** [reply]
I think you can just hold the key combo and let it do it's thing, but yes. A "Replace All" feature has been requested in #545 😃

**Reply by vkbo** [reply]
> When I figured out that I had to copy and paste the double quotes from the text. Entering it from the keyboard didn't work, so I guess they get converted to opening double quote and closing double quote somehow.

Yes, the quotes are auto-replaced by default. You can control that in Preferences and also pick which Quote symbols to replace with.

**Comment by peter88213** [comment]
If it is sufficient to perform the conversion in the exported ODT document afterwards, you might try [the "curly" extensions for LibreOffice](https://peter88213.github.io/curly/), which can also be used to adjust other typographical details such as apostrophes or dashes. 

In your case, you would use the "en-US" variant to convert the quotes into the exchange format and then format everything using the "en-GB" variant.
